packages/shared/package.json

```diff
@@ -42,7 +42,6 @@
     "dependencies": {
         "@deriv/translations": "^1.0.0",
         "@types/js-cookie": "^3.0.1",
-        "@types/moment": "^2.13.0",
```

Comment:
@niloo-fs is removal of “@types/moment” package intended? 🙂

Reply:
Yes, Since this lib deprecated I removed this

Reply:
For now I just discard this change, later I will create a card and will remove this lib in the whole project!

Reply:
For now I just discard this change, later I will create a card and will remove this lib in the whole project!